src/datasets/schemas/dataset.schema.ts

```diff
@@ -362,8 +362,8 @@ export class DatasetClass extends OwnableClass {
     description:
       "First name and last name of principal investigator(s). If multiple PIs are present, use a semicolon separated list. This field is required if the dataset is a Raw dataset.",
   })
-  @Prop({ type: String, required: false })
-  principalInvestigator?: string;
+  @Prop({ type: [String], required: false })
+  principalInvestigators?: string[];
```

Comment:
The description might also need to be changed?

Something like `Multiple PIs can be provided as separate strings in the array`?